web/src/beta/features/Dashboard/ContentsContainer/Projects/hooks.ts

```diff
@@ -239,13 +239,16 @@ export default (workspaceId?: string) => {
     async (event: React.ChangeEvent<HTMLInputElement>) => {
       const file = event.target.files?.[0];
       if (file) {
-        const result = await useImportProject(file);
+        const result = await useImportProject({
+          teamId: workspaceId || "",
```

Comment:
Sorry would need some help clarifying the reason we chose to use `workspaceId` as part of the input params. Since this function takes in the file from the user's local device as an input, I am wondering how the `workspaceId` will be used to improve the input. Is it so that we know which workspace the import is under?

Reply:
@mulengawilfred  the API requires to input which are teamId and file and the teamId is same as workspaceId you can check the importProject mutation

Reply:
My question is: is "" valid here? Or should we prevent the import if we don't get the workspaceId?

Reply:
Ah okay. I didn't see that. Thanks @mkumbobeaty  🙏